mbed_tools_ci_scripts/utils/package_helpers.py

```diff
@@ -0,0 +1,178 @@
+        """Gets package's name."""
+        return self._data.get('Name', UNKNOWN)
+
+    @property
+    def version(self) -> str:
+        """Gets package's version."""
+        return self._data.get('Version', UNKNOWN)
+
+    @property
+    def author(self) -> str:
+        """Gets package's author."""
+        return self._data.get('Author', UNKNOWN)
+
+    @property
+    def author_email(self) -> str:
+        """Gets package's author email."""
+        return self._data.get('Author-email', UNKNOWN)
+
+    @property
+    def licence(self) -> str:
+        """Gets package's licence."""
+        return self._data.get('License', UNKNOWN)
+
+    @property
+    def description(self) -> str:
+        """Gets package's licence."""
+        return self._data.get('Summary', UNKNOWN)
+
+    @property
+    def url(self) -> str:
+        """Gets package's URL."""
+        home_page = str(self._data.get('Home-page'))
+        if home_page:
+            return home_page
+        url = str(self._data.get('Project-URL'))
+        if url:
+            return url.split(',')[1].strip()
+        return UNKNOWN
+
+    def __str__(self) -> str:
+        """String representation."""
+        relevant_data = [
+            f'{getter}: {getattr(self, getter, None)}' for getter
+            in dir(self) if not getter.startswith("_") and getter
+        ]
+        return f'{", ".join(relevant_data)}'
+
+
+class ProjectMetadata:
+    """Metadata for a project."""
+
+    def __init__(self, package_name: str) -> None:
+        """Constructor."""
+        self._package_metadata: PackageMetadata = PackageMetadata(dict())
+        self._dependency_packages_metadata: List[PackageMetadata] = []
+        self._package_name: str = package_name
+
+    @property
+    def dependencies_metadata(self) -> List[PackageMetadata]:
+        """Gets all package's dependencies metadata."""
+        return self._dependency_packages_metadata
+
+    @property
+    def project_metadata(self) -> PackageMetadata:
+        """Gets project metadata."""
+        return self._package_metadata
+
+    @property
+    def package_name(self) -> str:
+        """Gets project's package name."""
+        return self._package_name
+
+    def __str__(self) -> str:
+        """String representation."""
+        dependencies_str = " | ".join(
+            [str(t) for t in self._dependency_packages_metadata])
+        project_str = f'Project [{self.package_name}]'
+        metadata_str = str(self._package_metadata)
+        other_str = f'Dependencies: [{dependencies_str}]'
+
+        return f'{project_str}: {metadata_str};  {other_str}'
+
+
+class ProjectMetadataParser:
+    """Parser of python package metadata."""
+    ENTRY_PATTERN = r'^([^:]*):(.*)$'
+
+    def __init__(self, package_name: str) -> None:
+        """Constructor."""
+        self._project_metadata: Optional[ProjectMetadata] = None
+        self._package_name = package_name
+
+    @staticmethod
+    def _get_metadata_lines(package: Any) -> Any:
```

Comment:
A default return of a list seems strange when the documented return type is Any.